seatunnel-core/seatunnel-core-starter/src/main/java/org/apache/seatunnel/core/starter/utils/ConfigShadeUtils.java

```diff
@@ -185,6 +198,19 @@ private static Config processConfig(
         return ConfigFactory.parseMap(configMap);
     }
 
+    public static Set<String> getSensitiveOptions(Config config) {
+        Set<String> sensitiveOptions =
+                new HashSet<>(
+                        TypesafeConfigUtils.getConfig(
+                                config != null && config.hasPath(Constants.ENV)
+                                        ? config.getConfig(Constants.ENV)
+                                        : ConfigFactory.empty(),
+                                SHADE_OPTIONS_OPTION,
+                                new ArrayList<>()));
+        sensitiveOptions.addAll(new ArrayList<>(Arrays.asList(DEFAULT_SENSITIVE_KEYWORDS)));
```

Comment:
we can directly use `sensitiveOptions.addAll(Arrays.asList(DEFAULT_SENSITIVE_KEYWORDS));`